controllers/networking-calico/charts/internal/calico/templates/calico.yaml

```diff
@@ -455,10 +455,10 @@ spec:
             - name: FELIX_DEFAULTENDPOINTTOHOSTACTION
               value: "ACCEPT"
             # Auto-detect the BGP IP address.
-           {{- if .Values.ipAutodetectionMethod }}
+           {{- if .Values.config.ipv4.autoDetectionMethod }}
```

Comment:
The reason for that is that the ipv4-related options went into the ipv4 dictionary where `pool` and `mode` are also unprefixed as wished by Rafael. 

Just as a sidenote: There is also an ipv6 equivalent option called `IP6_AUTODETECTION_METHOD`, which would go beneath ipv6 in case we need in the future.